cc1: warnings being treated as errors
elflint.c: In function 'check_dynamic':
elflint.c:1505: warning: initialized field overwritten
elflint.c:1505: warning: (near initialization for 'dependencies[2]')

--- elfutils-0.124/src/elflint.c.orig	2006-12-02 13:40:17.000000000 +0100
+++ elfutils-0.124/src/elflint.c	2006-12-02 13:40:29.929272750 +0100
@@ -1502,7 +1502,6 @@
       [DT_JMPREL] = { [DT_PLTRELSZ] = true, [DT_PLTREL] = true },
       [DT_RUNPATH] = { [DT_STRTAB] = true },
       [DT_PLTREL] = { [DT_JMPREL] = true },
-      [DT_PLTRELSZ] = { [DT_JMPREL] = true }
     };
   bool has_dt[DT_NUM];
   bool has_val_dt[DT_VALNUM];
